<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editarDialog</class>
 <widget class="QDialog" name="editarDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>&amp;Adicionar registo</string>
       </property>
       <property name="icon">
        <iconset resource="pixmaps/imagens.qrc">
         <normaloff>:/img/adicionar</normaloff>:/img/adicionar</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>&amp;Eliminar registo</string>
       </property>
       <property name="icon">
        <iconset resource="pixmaps/imagens.qrc">
         <normaloff>:/img/eliminar</normaloff>:/img/eliminar</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sairButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="pixmaps/imagens.qrc">
         <normaloff>:/img/exit</normaloff>:/img/exit</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pixmaps/imagens.qrc"/>
 </resources>
 <connections/>
</ui>
